File: Ejemplos_clase/Ejercicio_1/ejercicio1/Libro.ts
export class Libro{
    /*private _titulo: string;
    private _autor: string;
    private _isbn: string; 

    
    constructor(titulo: string, autor: string, isbn: string;){
        this._titulo = titulo;
        this._autor = autor;
        this._isbn = isbn;
    }*/
    
    constructor(
    private _titulo: string,
    private _autor: string,
    private _isbn: string
    ){}
    get titulo() { return this._titulo}
    get autor() { return this._autor}
    get isbn() { return this._isbn}
    
}

File: Ejercicio1_biblioteca/Libro.java
/**
 * 
 */
/**
 * 
 */
public class Libro {
	private String titulo;
    private Autor autor;
    private String isbn;

    public Libro(String titulo, Autor autor, String isbn) {
        this.titulo = titulo;
        this.autor = autor;
        this.isbn = isbn;
    }

    public String getTitulo() { 
    	return titulo; 
    }
    public Autor getAutor() { 
    	return autor; 
    }
    public String getIsbn() {
    return isbn; 
    }
 }
